packages/typespec-ts/test/modularIntegration/generated/lro_modular/generated/src/api/operations.ts

```diff
@@ -0,0 +1,193 @@
+    context,
+    pollerOptions
+  )) as SimplePollerLike<OperationState<User>, User>;
+  return poller;
+}
+
+export interface DeleteOptions extends RequestOptions {}
+
+export function _deleteOperationSend(
+  context: Client,
+  name: string,
+  options: DeleteOptions = { requestOptions: {} }
+): StreamableMethod<
+  DeleteOperation202Response | DeleteOperationDefaultResponse
+> {
+  return context
+    .path("/azure/core/lro/standard/users/{name}", name)
+    .delete({
+      allowInsecureConnection: options.requestOptions?.allowInsecureConnection,
+      skipUrlEncoding: options.requestOptions?.skipUrlEncoding,
+      headers: { ...options.requestOptions?.headers },
+    });
+}
+
+export async function _deleteOperationDeserialize(
+  result: DeleteOperation202Response | DeleteOperationDefaultResponse
+): Promise<OperationStatus> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return {
+    id: result.body["id"],
+    status: result.body["status"],
+    error: !result.body.error ? undefined : result.body.error,
+  };
+}
+
+/** Deletes a User */
+/**
+ *  @fixme delete is a reserved word that cannot be used as an operation name. Please add @projectedName(
+ *       "javascript", "<JS-Specific-Name>") to the operation to override the generated name.
+ */
+export async function beginDelete(
+  context: Client,
+  name: string,
+  options: DeleteOptions = { requestOptions: {} }
+): Promise<SimplePollerLike<OperationState<OperationStatus>, OperationStatus>> {
+  const pollerOptions = {
+    requestMethod: "DELETE",
+    requestUrl: "/azure/core/lro/standard/users/{name}",
+    deserializeFn: _deleteOperationDeserialize,
+    sendInitialRequestFn: _deleteOperationSend,
+    sendInitialRequestFnArgs: [context, name, options],
+  } as GetLongRunningPollerOptions;
+  const poller = (await getLongRunningPoller(
+    context,
+    pollerOptions
+  )) as SimplePollerLike<OperationState<OperationStatus>, OperationStatus>;
+  return poller;
+}
+
+export interface ExportOptions extends RequestOptions {}
+
+export function _exportSend(
+  context: Client,
+  name: string,
+  format: string,
+  options: ExportOptions = { requestOptions: {} }
+): StreamableMethod<Export202Response | ExportDefaultResponse> {
+  return context
+    .path("/azure/core/lro/standard/users/{name}:export", name)
+    .post({
+      allowInsecureConnection: options.requestOptions?.allowInsecureConnection,
+      skipUrlEncoding: options.requestOptions?.skipUrlEncoding,
+      headers: { ...options.requestOptions?.headers },
+      queryParameters: { format: format },
+    });
+}
+
+export async function _exportDeserialize(
+  result: Export202Response | ExportDefaultResponse
+): Promise<ExportedUser | undefined> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  if (!result.body.result) {
+    return undefined;
+  }
+
+  return {
+    name: result.body.result["name"],
+    resourceUri: result.body.result["resourceUri"],
```

Comment:
result.bady.result looks weird.

Reply:
It is one of LRO standard cases, https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-operations

![image](https://github.com/Azure/autorest.typescript/assets/9943211/bbfa7f9f-9d70-4ad0-b7fe-ee3d15d33698)